src/NuGet.Core/NuGet.Packaging/Strings.Designer.cs

```diff
@@ -222,6 +222,15 @@ internal class Strings {
             }
         }
         
+        /// <summary>
+        ///   Looks up a localized string similar to The package doesn&apos;t contain {0} file: {1}.
+        /// </summary>
+        internal static string Error_MissingRequiredFiles {
```

Comment:
Why are you adding a generic message? Will it be used anywhere else?

Reply:
Created a generic error message just in case if we happen to have a scenario in the future where we need to notify user of missing required file. As of today, {0} - is for .nuspec string and {1} - is the package folder.

Reply:
The nuspec is the only required file in a package. 

And that won't change, cause we need to backwards compatible :)